corehq/apps/reports/standard/monitoring.py

```diff
@@ -323,36 +271,105 @@ def add_numeric_cell(text, value=None):
+                filters.term('closed', False))
+        )
+
+        landmarks_aggregation = self._landmarks_aggregation(end_date)
+
+        top_level_aggregation = TermsAggregation('users', 'user_id')\
+            .aggregation(landmarks_aggregation)\
+            .aggregation(active_total_aggregation)\
+            .aggregation(inactive_total_aggregation)
+
+        query = case_es.CaseES().domain(self.domain).user(users_by_id.keys())
+        query = query.aggregation(top_level_aggregation)
+        return query.run()
+
+    def _landmarks_aggregation(self, end_date):
+        landmarks_aggregation = RangeAggregation('landmarks', 'modified_on')
+        for landmark in self.landmarks:
+            start_date = ServerTime(self.utc_now - landmark).phone_time(self.timezone).done()
+            landmarks_aggregation.add_range(AggregationRange(start_date, end_date, key=landmark.days))
+        landmarks_aggregation.aggregation(FilterAggregation('active', filters.term('closed', False)))
+        landmarks_aggregation.aggregation(FilterAggregation('closed', filters.term('closed', True)))
+        return landmarks_aggregation
+
+    class Row(object):
+        def __init__(self, report, user, bucket):
+            self.report = report
+            self.user = user
+            self.bucket = bucket
+            if bucket:
+                self.landmarks = bucket.landmarks.buckets_dict
+
+        def active_count(self, landmark_key):
+            return 0 if not self.bucket else self.landmarks[landmark_key].active.doc_count
+
+        def modified_count(self, landmark_key):
+            return 0 if not self.bucket else self.landmarks[landmark_key].doc_count
+
+        def closed_count(self, landmark_key):
+            return 0 if not self.bucket else self.landmarks[landmark_key].closed.doc_count
+
+        def total_inactive_count(self):
+            return 0 if not self.bucket else self.bucket.inactive_total.doc_count
+
+        def total_active_count(self):
+            return 0 if not self.bucket else self.bucket.active_total.doc_count
+
+        def header(self):
+            return self.report.get_user_link(self.user)
+
+    class TotalRow(object):
+        def __init__(self, rows, header):
+            self.rows = rows
+            self._header = header
+
+        def active_count(self, landmark_key):
+            return sum([row.active_count(landmark_key) for row in self.rows])
+
+        def total_inactive_count(self):
+            return sum([row.total_inactive_count() for row in self.rows])
+
+        def total_active_count(self):
+            return sum([row.total_active_count() for row in self.rows])
+
+        def modified_count(self, landmark_key):
+            return sum([row.modified_count(landmark_key) for row in self.rows])
+
+        def closed_count(self, landmark_key):
+            return sum([row.closed_count(landmark_key) for row in self.rows])
+
+        def header(self):
+            return self._header
```

Comment:
Yea, this report is pretty hard to figure out. Once product knows what they want it should be easy to fix.